packages/js/src/utils/scope.ts

```diff
@@ -0,0 +1,26 @@
+export enum ScopeValue {
+  name = 'name',
+  email = 'email',
+  offline_access = 'offline_access',
+  openid = 'openid',
+}
+
+export const DEFAULT_SCOPE_VALUES = [ScopeValue.openid, ScopeValue.offline_access];
+export const DEFAULT_SCOPE_STRING = DEFAULT_SCOPE_VALUES.join(' ');
```

Comment:
I think @gao-sun once said we don't have convension to declare upper snake case variables, would be just fine to use camel case here

Reply:
- upper snake case for global constants
- camel case for local constants

Reply:
OK, based on the latest team discussion, we should use camel case name for all constants for js sdk